framework/src/modules/http_api/httpApi.js

```diff
@@ -0,0 +1,95 @@
+const { createLoggerComponent } = require('../../components/logger');
+const { createCacheComponent } = require('../../components/cache');
+const { createStorageComponent } = require('../../components/storage');
+const { createSystemComponent } = require('../../components/system');
+const {
+	bootstrapStorage,
+	setupServers,
+	startListening,
+	subscribeToEvents,
+	bootstrapSwagger,
+	bootstrapCache,
+} = require('./init_steps');
+
+module.exports = class HttpApi {
+	constructor(channel, options) {
+		options.config.root = __dirname; // TODO: See wy root comes defined for the chain module.
+		this.channel = channel;
+		this.options = options;
+		this.logger = null;
+		this.scope = null;
+	}
+
+	async bootstrap() {
+		global.constants = this.options.constants;
```

Comment:
overwriting the global constants seems dangerous here.
is it ok?

Reply:
This issue is solved by Nazars PR on refactoring configuration files. Will follow after this one.